### PR TITLE
LIVE-2842 Osmosis send/receive/staking in LLD + send/receive in LLM

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

This PR adds the support of Osmosis integration.
- LLD : send/receive + staking flows
- LLM : send/receive only

### ❓ Context

- **Impacted projects**: `live-common`, `live-desktop`, `live-mobile`
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-2842

For more info please check : https://github.com/LedgerHQ/ledger-live/pull/155 and https://github.com/LedgerHQ/ledger-live/pull/723

### ✅ Checklist

- [x] **Test coverage**: _Did you write any tests to cover the changes introduced by this pull request?_
- [x] **Atomic delivery**: _Is this pull request standalone? In order words, does it depend on nothing else?_
- [x] **No breaking changes**: _Does this pull request contain breaking changes of any kind? If so, please explain why._

### 🚀 Expectations to reach

- [x] Integration of Osmosis in LLD : add accounts / sync / send / receive / staking flows (same ones as cosmos).
- [x] Integration of Osmosis in LLM : add accounts / sync / send / receive
- [x] LLD integration is feature flagged

### CODE REVIEW

**Comment by gre** [comment]
Checklist on the PR description should be checked, if they can't add an explanation. if this is still a WIP, make the PR a Draft 🙏

**Review by lambertkevin — APPROVED** [review]
Just one thing about the documentation that can be updated later. 
Still a lot of questions regarding the Product team though.

File: .github/workflows/bot-coin-osmosis.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+name: Bot 'osmosis on Silicium'
```
Comment (valpinkman) [inline .github/workflows/bot-coin-osmosis.yml:1]:
```suggestion
name: [Bot] Osmosis on Silicium
```

**Review by valpinkman — CHANGES_REQUESTED** [review]
new naming convention for jobs ;)

**Comment by gre** [comment]
is it the same as #745  ?

**Comment by haammar-ledger** [comment]
:information_source: **This should be merged**, last CI failure is an unrelated known issue on LLD UI tests, currently affecting all PRs.

File: libs/ledger-live-common/package.json [hunk 2]
```diff
@@ -116,12 +116,15 @@
...
     "@cosmjs/crypto": "^0.26.5",
+    "@cosmjs/launchpad": "^0.26.5",
+    "@cosmjs/ledger-amino": "^0.26.5",
     "@cosmjs/proto-signing": "^0.26.5",
     "@cosmjs/stargate": "^0.26.5",
     "@crypto-com/chain-jslib": "0.0.19",
     "@ethereumjs/common": "^2.6.2",
     "@ethereumjs/tx": "^3.5.0",
     "@hashgraph/sdk": "^2.10.1",
+    "@keplr-wallet/cosmos": "^0.9.16",
```
[the remark this answers is not in the document]
Comment (gre) [reply libs/ledger-live-common/package.json:127]: was this tackled?

[the remark this answers is not in the document]
Comment (haammar-ledger) [reply libs/ledger-live-common/package.json:127]: Now yes :+1::slightly_smiling_face:

**Comment by haammar-ledger** [comment]
:information_source: **This can be merged, CI failure on [Test] Desktop App (windows) is an unrelated known flakiness issue.**

**Review by gre — COMMENTED** [review]
it's the first time I'm aware of a coin implementation that depends on another.
did we project this topic in context of future coin modularisation? It may be problematic.
makes me wonder why we didn't consider osmosis to live in the cosmos family?

**Pull request merged**